libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -342,6 +344,16 @@ dictionary LspInformation {
     OpeningFeeParamsMenu opening_fee_params_list;
 };
 
+dictionary OpenChannelFeeRequest {
+    u64 amount_msat;
+    u32? expiry;
```

Comment:
The expiry in ReceivePaymentRequest should also be u32 then?

Reply:
Yes, based on @JssDWt feedback I changed that to u32 and that made my apply the change also on ReceivePaymentRequest at the same commit to be consistent.